### PR TITLE
Let footer change with theme, etc

### PR DESCRIPTION
**Pull request by henryyang42**
- Change some static file to use minified one
- Let footer change with theme
- Adjust navbar shrink size (so that it won't shrink on my 13'' NB)

### CODE REVIEW

**Comment by nickelcheng** [comment]
@bruce3557 please check

File: index/templates/index/base.html [hunk 1]
```diff
@@ -192,16 +192,19 @@
...
-        </p>
-    </div>
+    <footer>
+      <div class="navbar navbar-inverse navbar-fixed-bottom sticky-footer">
+        <div class="container-fluid">
+          <div class="navbar-collapse collapse">
+            <button id="contact_us" data-clipboard-text="nthucsoj@gmail.com" title="Click to copy me.">
+              Contact us
+            </button>
+            <script src={% static "js/clipboard.js"%}></script>
```
Comment (bruce3557) [inline index/templates/index/base.html]: May I ask why we need this here?

File: index/templates/index/base.html [hunk 2]
```diff
@@ -48,7 +48,7 @@
     <link rel="stylesheet" href={% static "css/index.css" %} media="screen">
     <link rel="stylesheet" href={% static "css/mfp-fade.css" %} media="screen">
     <script src={% static "js/clock.js" %}></script>
-    <script src={% static "zeroclipboard/dist/ZeroClipboard.js"%}></script>
+    <script src={% static "zeroclipboard/dist/ZeroClipboard.min.js"%}></script>
```
Comment (bruce3557) [inline index/templates/index/base.html]: Please load js library at the end of html to boost performance

File: index/templates/index/base.html (hunk 1 above)
Comment (henryyang42) [inline index/templates/index/base.html]: I don't know either. @drowsy810301

File: index/templates/index/base.html (hunk 2 above)
Comment (bruce3557) [inline index/templates/index/base.html]: http://robertnyman.com/2008/04/23/where-to-include-javascript-files-in-a-document/

**Comment by henryyang42** [comment]
Put js to bottom now. But I remain ``jquery`` on top because it's used everywhere. Moving it down will cause a lot of error.

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**